python/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Jorge Leitao
```

Comment:
This presumably needs to change to an ASF copyright?

Reply:
Also, I assume we need to licence under ASL 2.0 as well

Reply:
I'm unsure whether we can use an MIT license in an ASF project or not and wasn't able to find answers right away on the ASF site but we can ask for advice on this from the incubator folks.

What is the reason for the MIT license here? Is this something that is expected in the Python ecosystem?

Reply:
This would all change to Apache 2.0. It was just the license I had up to this point in jorgecaleitao. No need for MIT here.

Note that I am the sole copyright holder of this code, so I can licensed it in any way I want to ASF. I will need to push a commit to change this to Apache 2.0, and add headers everywhere as per requirements, just wanted to get the ball rolling ^_^